NOTICE

```diff
@@ -1,17 +1,44 @@
+== Notices and Attribution ==
 This product includes software developed by Google
  Snappy: http://code.google.com/p/snappy/ (New BSD License)
 
 This product includes software developed by Apache
  PureJavaCrc32C from apache-hadoop-common http://hadoop.apache.org/
  (Apache 2.0 license)
 
-This library containd statically linked libstdc++. This inclusion is allowed by 
-"GCC RUntime Library Exception" 
+This library contains statically linked libstdc++. This inclusion is allowed by 
+"GCC Runtime Library Exception" 
 http://gcc.gnu.org/onlinedocs/libstdc++/manual/license.html
 
 == Contributors ==
-  * Tatu Saloranta  
+  * Tatu Saloranta
     * Providing benchmark suite
   * Alec Wysoker
     * Performance and memory usage improvement
 
+== Export Control Notice ==
+This distribution includes cryptographic software. The country in
+which you currently reside may have restrictions on the import,
+possession, use, and/or re-export to another country, of
+encryption software. BEFORE using any encryption software, please
+check your country's laws, regulations and policies concerning the
+import, possession, or use, and re-export of encryption software, to
+see if this is permitted. See <http://www.wassenaar.org/> for more
+information.
+
+The U.S. Government Department of Commerce, Bureau of Industry and
+Security (BIS), has classified this software as Export Commodity
+Control Number (ECCN) 5D002.C.1, which includes information security
+software using or performing cryptographic functions with asymmetric
+algorithms. The form and manner of this Apache Software Foundation
+distribution makes it eligible for export under the License Exception
+ENC Technology Software Unrestricted (TSU) exception (see the BIS
+Export Administration Regulations, Section 740.13) for both object
+code and source code.
+
+The following provides more details on the included cryptographic software:
+
+This software uses the SSL libraries from the Jetty project written
+by mortbay.org.
+Hadoop Yarn Server Web Proxy uses the BouncyCastle Java
+cryptography APIs written by the Legion of the Bouncy Castle Inc.

```

Comment:
I think this is a copy from Hadoop's notice file, but snappy-java's jar itself doesn't contain any of Hadoop products as a dependency, so this statement unnecessarily scares jar-only users of snappy-java.

This notice applied only to the source/test code, not to the published jar files. I think adding a pointer to the Hadoop's NOTICE file is more appropriate here than copying everyt sentence.